macros.cmake

```diff
@@ -76,7 +76,7 @@ macro(download_contrib_archive libname)
   # constant
   # Currently this points to an FTP at FU Berlin
   # Sources are checked out regularly from OpenMS/contrib-sources via a cron job
-  set(_BASE_URL "http://ftp.mi.fu-berlin.de/pub/OpenMS/contrib-sources/")
+  set(_BASE_URL "http://vmi79711.contabo.host:666/OpenMS/contrib/")
```

Comment:
Ok, but this will yield a 404 right now. It needs intervention from the contib repo maintainer. Will do that though.